build.xml

```diff
@@ -263,7 +289,9 @@
        <mkdir dir="${dist}"/>
        <mkdir dir="${dist.tmp}"/>
        <unjar dest="${dist.tmp}">
-           <fileset dir="${lib}" />
+           <fileset dir="${lib}">
+               <exclude name="**/jacocoant.jar"/> <!-- must exclude this jar from packing into picard - otherwise build fails -->
```

Comment:
I'd change the comment. The truth is that we do not want jacoco packed together with picard...right?